paper.tex

```diff
@@ -647,11 +647,11 @@ \subsection{GPU: Execution Time} \label{subsection:gpu-results}
 
 \subsection{GPU: Input Processing} \label{subsection:encoding-results}
 
-Figure~\ref{fig:tag-monoid-gpu-timings} shows the run time of the tag monoid parallel prefix sums with increasing input size. The prefix sums scale better at large input sizes compared to our stroke expansion kernel. This can be attributed to a lack of expensive computations and high control-flow uniformity in the tag monoid kernel. The CPU encoding times (prior to the GPU prefix sums) for some of the large scenes is shown in Table~\ref{table:bump-estimate-cost}. Dashing on the GPU was out of scope, so the dash style applied to the \emph{long dash} scene had to be processed on the CPU and dashes were sequantially encoded as individual strokes. Including CPU-side dashing, the time to encode \emph{long dash} on the Apple M1 Max was approximately 25 ms on average, while the average time to encode the path after dashing was 4.14 ms.
+Figure~\ref{fig:tag-monoid-gpu-timings} shows the run time of the tag monoid parallel prefix sums with increasing input size. The prefix sums scale better at large input sizes compared to our stroke expansion kernel. This can be attributed to a lack of expensive computations and high control-flow uniformity in the tag monoid kernel. The CPU encoding times (prior to the GPU prefix sums) for some of the large scenes is shown in Table~\ref{table:bump-estimate-cost}. Our implementation still applies dashing on the CPU. It is conceptually fairly straightforwad to implement on GPU; the core algorithm is prefix sum applied to segment length, and the Euler spiral representation is particularly well suited. However, there are complex implementatoin tradeoffs, so the dash style applied to the \emph{long dash} scene had to be processed on the CPU and dashes were sequentially encoded as individual strokes. Including CPU-side dashing, the time to encode \emph{long dash} on the Apple M1 Max was approximately 25 ms on average, while the average time to encode the path after dashing was 4.14 ms.
```

Comment:
I have a few thoughts on this paragraph:

AFAICT the paper still doesn't make any mention of dashing until this section. I think dashing should be defined earlier in the introduction alongside the discussion of cap and join styles. The part where we make the strong correctness claim should probably say that we didn't implement dashing, so that the scope of the paper is fully laid out upfront.

-------
Apart from that some minor suggestions:
> Our implementation still applies dashing on the CPU.

I would drop the "still".

> However, there are complex implementatoin tradeoffs, so the dash style applied to the \emph{long dash} scene had to be processed on the CPU and dashes were sequentially encoded as individual strokes.

There is one typo here (implementatoin). Aside from that, the sentence claims there are trade-offs without really explaining them, so it feels a bit hand-wavy, and the second part of the sentence doesn't clearly follow from that. I think we should either expand on those trade-offs or more openly state that we haven't fully explored them.